app/src/main/kotlin/cz/covid19cz/erouska/ext/Context.kt

```diff
@@ -22,35 +24,18 @@ import cz.covid19cz.erouska.ui.base.BaseFragment
 import cz.covid19cz.erouska.utils.CustomTabHelper
 import cz.covid19cz.erouska.utils.L
 
+fun Context.hasBle(c: Context): Boolean {
```

Comment:
To není potřeba kontrolovat ne? Play Store by neměl dovolit instalaci tam kde to není.

Reply:
asi máš pravdu, jsou to refactorované pozůstatky z legacy verze